app/v2/controllers/V2MovementsController.scala

```diff
@@ -68,6 +67,7 @@ trait V2MovementsController {
   def getMovement(movementType: MovementType, movementId: MovementId): Action[AnyContent]
   def getMovements(movementType: MovementType, updatedSince: Option[OffsetDateTime], movementEORI: Option[EORINumber]): Action[AnyContent]
   def attachMessage(movementType: MovementType, movementId: MovementId): Action[Source[ByteString, _]]
+  def attachLargeMessage(movementId: MovementId, messageId: MessageId): Action[Source[ByteString, _]]
```

Comment:
tests are missing for attachLargeMessage

Reply:
Will do